releasing/README.md

```diff
@@ -410,267 +429,7 @@ https://github.com/kubernetes/kubernetes/pull/103419
-
-```
-tag="${module}/v${major}.${minor}.${patch}"
-echo "tag=$tag"
-```
-
-### Pin modules to their dependencies.
-
-This is achieved via a `replace` directive
-in a module's `go.mod` file.
-
-```
-# Update the following as needed, obviously.
-
-# git checkout -b pinTheRelease
-# go mod edit -dropreplace=sigs.k8s.io/kustomize/api    $module/go.mod
-# go mod edit -require=sigs.k8s.io/kustomize/api@v0.1.1 $module/go.mod
-# git commit -a -m "Drop API module replacement"
-
-```
-
-### Push the release branch
-
-```
-git push -f upstream $branch
-```
-
-#### if replacing a release...
-
-Must delete the tag before re-pushing it.
-Dangerous - only do this if you're sure nothing
-has already pulled the release.
-
-Delete the tag locally:
-
-```
-git tag --delete $tag
-```
-
-Delete it upstream:
-```
-# Disable push protection:
-git remote set-url --push upstream git@github.com:kubernetes-sigs/kustomize.git
-
-# The empty space before the colon effectively means delete the tag.
-git push upstream :refs/tags/$tag
-
-# Enable push protection:
-git remote set-url --push upstream no_push
-```
-
-Optionally visit the [release page] and delete
-(what has now become) the _draft_ release for that
-version.
-
-### Tag the local repository
-
-```
-git tag -a $tag -m "Release $tag on branch $branch"
-```
-
-Move the `latest_kustomize` tag:
-```
-git tag -d latest_kustomize
-git push upstream :latest_kustomize
-git tag -a latest_kustomize
-```
-
-### Optionally build locally
-
-[localbuild.sh]: localbuild.sh
-
-Load the same version of `goreleaser` referenced in `cloudbuild.yaml` via docker and run [localbuild.sh] from the container's command line:
-
-```
-# Get goreleaser image from cloudbuild.yaml 
-export GORELEASER_IMAGE=goreleaser/goreleaser:v0.172.1
-
-# Drop into a shell
-docker run -it --entrypoint=/bin/bash  -v $(pwd):/go/src/github.com/kubernetes-sigs/kustomize -w /go/src/github.com/kubernetes-sigs/kustomize $GORELEASER_IMAGE
-
-# Run build
-./releasing/localbuild.sh TAG [--snapshot]
-```
-
-
-### Optionally build and release locally
-
-[cloudbuild-local.sh]: cloudbuild-local.sh
-
-Install [`cloud-build-local`], then run [cloudbuild-local.sh]:
-
-```
-./releasing/cloudbuild-local.sh $module
-```
-
-This should create release artifacts in a local directory.
-
-### Trigger the cloud build by pushing the tag
-
-Push the tag:
-
-```
-git remote set-url --push upstream git@github.com:kubernetes-sigs/kustomize.git
-git push upstream $tag
-git push upstream latest_kustomize
-git remote set-url --push upstream no_push
-```
-
-This triggers [Google Cloud Build].
 
-### Update release notes
+# Testing changes to the release pipeline
 
-Visit the [release page] and edit the release notes as desired.
+You can test the release script locally by running [cloudbuild.sh](cloudbuild.sh) in a container or by installing Cloud Build Local and running [cloudbuild-local.sh](cloudbuild-local.sh). See each of those files for more details on their usage.
```

Comment:
I tried this, but I couldn't figure out how to set the value of $TAG_NAME. Regardless, it seemed to be running the steps correctly and threw the exact error I expected given an empty $TAG_NAME so I think it does work.

Reply:
Should just be the first arg, e.g. `./releasing/cloudbuild-local.sh kyaml/v0.13.4`

Reply:
Ah thanks! I missed the comment at the top of cloudbuild-local.sh.

Now I get the output:

```
ERROR: build step 3 "goreleaser/goreleaser:v0.179.0" failed: Failed to decrypt "GITHUB_TOKEN" using key "projects/k8s-staging-kustomize/locations/global/keyRings/kust-cloud-key-ring/cryptoKeys/kust-cloud-key-name": googleapi: Error 403: Permission 'cloudkms.cryptoKeyVersions.useToDecrypt' denied on resource 'projects/k8s-staging-kustomize/locations/global/keyRings/kust-cloud-key-ring/cryptoKeys/kust-cloud-key-name' (or it may not exist)., forbidden
```

which I assume is because I don't have keyring decryption permissions on the google cloud project (as expected), so this LGTM.